JavaErickHernandez/Day3/RansomNote.java

```diff
@@ -0,0 +1,87 @@
+package bootcamp;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.Map;
+import java.util.Scanner;
+
+public class RansomNote {
+	static String wordsMagazine = "Enter the number of words in the magazine: ";
+	static String wordsNote = "Enter the number of words in the note: ";
+	static String word = "Enter the word: ";
+	static String yes = "YES";
+	static String no = "NO";
+	static String tryAgainMessage = "Try Again";
+	static HashSet<String> noteWords = new HashSet<String>();
+	static HashSet<String> magazineWords = new HashSet<String>();
```

Comment:
When you variable is a Collection, it is good practice put name in singular and them the the type, example: tableList, nameSet, etc...